examples/create-window/main.go

```diff
@@ -103,5 +105,40 @@ func main() {
 		if xerr != nil {
 			fmt.Printf("Error: %s\n", xerr)
 		}
+
+		// This is how accepting events work:
+		// The application checks what event we got
+		// (the event must be registered using either xproto.CreateWindow (see l.35) or
+		//  xproto.ChangeWindowAttributes (see l.50)).
```

Comment:
Please reword like this
```
The application checks what event we got and reacts to it accordingly. All events are defined in the xproto subpackage and the event we wan't to handle must be registered using either xproto.CreateWindow or xproto.ChangeWindowAttributes before.
```

Why? I don't think it's good practice to mention lines in comments. After changes you'll have to check comments if lines are matching and make fixes. Also too long text in parenthesis and nested () are a distraction.

Reply:
c3652e2